ios/fastlane/Fastfile

```diff
@@ -198,6 +198,56 @@ platform :ios do
     )
   end
 
+  desc "Submit a new Beta Build to TestFlight, using CircleCI"
+  lane :beta_circleci_testflight do |options|
+
+    # Setup the circleci environment
+    setup_ci(
+      provider: "circleci"
+    )
+
+    # use the App Store API Key to authenticate
+    api_key = app_store_connect_api_key(
+      key_id: "#{ENV['APP_STORE_API_KEY_ID']}",
+      issuer_id: "#{ENV['APP_STORE_API_KEY_ISSUER_ID']}",
+      key_filepath: "./fastlane/AuthKey_#{ENV['APP_STORE_API_KEY_ID']}.p8",
+      duration: 1200,
+    )
+
+    sync_code_signing(type: "appstore", api_key: api_key)
+
+    # Install pods
+    cocoapods
+
+    # build the app
+    gym(
+      scheme: "ItaliaApp",
+      clean: true,
+      export_method: "app-store",
+      workspace: "ItaliaApp.xcworkspace",
+      include_symbols: false,
+      # sdk: "iphoneos8.4"
+    )
+
+    previous_tag = sh("git tag --sort=-taggerdate | grep \\\\-rc\\\\. | sed -n '2 p' | tr -d '\n'")
+
+    # generate the changelog based on last commits
+    changelog = changelog_from_git_commits(
+      tag_match_pattern: "*" +previous_tag + "*" ,
+      pretty:"- %s",
+      merge_commit_filtering: "exclude_merges"
+    )
+
+    # upload to App store
+    pilot(
+      api_key: api_key,
+      changelog: changelog,
+      distribute_external: true,
+      notify_external_testers: true,
+      groups: ["Amici&Familiari", "PagoPa"],
```

Comment:
Not sure how often we plan to change the groups: it might be worth to get them via envvar.

Reply:
My idea here was to eventually implement the groups as parameter (eg. send the new beta to some additional external groups), if deemed useful, in the story https://pagopa.atlassian.net/browse/IAI-6, that will allows to trigger the workflow using, for example, the github actions and also add additional parameters (eg. custom message to send to slack channel). With the actual implementation, the workflow is triggered only by the tag and we have no way to pass further parameters. 
What you think, could be ok to add this feature (additional group) in the IAI-6 story?
Btw at the moment is always possible to add other groups by the App Store Connect interface at any time.